### PR TITLE
Update showpage

### PR DESCRIPTION
**Pull request by hidehiro98**
🙏 @db0sch

### CODE REVIEW

**Comment by db0sch** [comment]
When you have a commit like `fix minor change`, you should rebase your branch.
With `rebase -i HEAD~4` (4 being the number of commit in this branch.
Then, you can `squash` (instead of `pick`) the commit, and it will merge it with the previous commit.

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -1,2 +1,45 @@
 module ApplicationHelper
+  def comment_time(time)
```
Comment (db0sch) [inline app/helpers/application_helper.rb]: this should be in the `CommentHelper` i think.

**Review by db0sch — CHANGES_REQUESTED** [review]
Just one file to change.

**Comment by hidehiro98** [comment]
@db0sch 
Thank you very much for the advice of the rebase branch!!
And I moved helper method to the new helper file.

**Pull request merged**